README.md

```diff
@@ -42,24 +44,21 @@ Run the sample
 
 - Get the `client_secret_NNNN.json`
   [OAuth client ID][oauth2-installed-app] JSON file for the application from
```

Comment:
Since the instructions to generate this file are not inline, it would be preferred to link folks to the docs that describe creating new OAuth credentials.